lua/wfp_verify.lua

```diff
@@ -75,14 +75,15 @@ function verifyWFP(tTarget, tWfpControl, iChiptype, atWfpConditions, tPlugin, tF
                 end
             else
 				-- new entry for the current flash
+
                 if atFlashDataTable[strChunkKey] == nil then
                     atFlashDataTable[strChunkKey] = {}
```

Comment:
Hint: you could also write
atFlashDataTable[strChunkKey] = {
	atChunkList = {},
	tBus = tBus, 
	ulUnit = ulUnit,
	ulChipSelect = ulChipSelect
}